element-template-generator/http-dsl/src/main/java/io/camunda/connector/generator/dsl/http/HttpOperationProperty.java

```diff
@@ -33,8 +34,19 @@ public static HttpOperationProperty createStringProperty(
   }
 
   public static HttpOperationProperty createEnumProperty(
-      String id, Target target, String description, boolean required, List<String> choices) {
-    return new HttpOperationProperty(id, target, description, required, Type.ENUM, choices, null);
+      String id, Target target, String description, boolean required, List<Object> choices) {
```

Comment:
List<Object> choices => List<?> choices, then we can delete the casts (see my next comments)